### ISSUE TITLE
Not able to get access to kubectl after having logged in and logged out the second time.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#9515]
Ranche server version - v1.6.6-rc5

openLDAP enabled setup with Kubernetes environment.

2 users , user1 and usrer2 have access to this environment.

In fresh browser session:
1. log in as user1
2. Launch kubectl shell and executing any kubectl command succeeds.
3. logout
4. log in as user2
5. Launch kubectl shell and executing any kubectl command succeeds.
6. logout
7. log in as user1.
8. Launch kubectl shell and try to execute any kubectl command. This fails.

```
> kubectl get nodes
error: You must be logged in to the server (the server has asked for the client to provide credentials)
>
```

I am able log in successfully using user1 and get access to kubectl commands when I try it in a different browser.

Seems like there some token cached which prevents access to kubectl.

**Comment by deniseschannon** [comment]
I was able to reproduce the problem, but didn't hit it immediately after logging into the UI with the first user. After a couple of minutes, I no longer had access. 

It seems that some token in the UI expires and then if you refresh the UI, it will work again.

**Comment by soumyalj** [comment]
This issue was seen after an upgrade to v1.6.6-rc5

**Comment by deniseschannon** [comment]
Available in v1.6.8-rc5

**Comment by sangeethah** [comment]
Tested with v1.6.8-rc5.

Able to get access to kubectl after having logged in and logged out as different users.
